### ISSUE TITLE
feature request, print success status as last thing in stdout log

### ISSUE THREAD

**Issue by schultzm** [tseemann/nullarbor#133]
At present, the last log to be printed to screen is the output from `make space`.  Without scrollback, it is not easy to see if the run finished without errors.  It might be nice to have `success` or `fail` status at the end of the output...

**Comment by tseemann** [comment]
Yes I agree this is annoying. I might suppress the `rm` lines with `@rm` in the Makefile

You chould run it like `make ........ | tee nullarbor.log` so it saves the log file.

Also, the fail success will be in the error code in the shell.  eg `echo $?`  (0=good)
